ComplexityAnalysis/SpaceComplexity.md

```diff
@@ -40,7 +40,7 @@ Space complexity = Input Size + Auxillary Space.
 
 ## To design the algorithm
  
-Time complexity much be lower, where the space complexity can be higher.
+ Time complexity is inversely proportional to the space.so the time complexity can be lower if the space complexity is higher.

```

Comment:
Change the second sentence to start with the Capital case and give a space after '.' like '. So'